### PR TITLE
Return the renderer and view created when statefulCallClient.createView is called

### PR DESCRIPTION
**Pull request by JamesBurnside**
# What
* Return the renderer and view created when statefulCallClient.createView is called

Next PRs in stack:
* #1890
* #1891

# Why
Currently we throw away the view so the likes of view.updateScalingMode cannot be called.
See next PR in stack for our stateful and components making use of this.

# How Tested
See final PR in stack: #1891

### CODE REVIEW

File: packages/calling-stateful-client/review/calling-stateful-client.api.md [hunk 1]
```diff
@@ -173,7 +175,7 @@ export interface RemoteVideoStreamState {
 
 // @public
 export interface StatefulCallClient extends CallClient {
-    createView(callId: string | undefined, participantId: CommunicationIdentifier | undefined, stream: LocalVideoStreamState | RemoteVideoStreamState, options?: CreateViewOptions): Promise<void>;
```
Comment (JamesBurnside) [inline packages/calling-stateful-client/review/calling-stateful-client.api.md:176]: Is this a breaking change? Should I do `Promise<void | VideoStreamRendererAndView` instead?

Comment (prprabhu-ms) [reply packages/calling-stateful-client/review/calling-stateful-client.api.md:176]: You are good: https://www.typescriptlang.org/play?#code/PTAEFcDsBMFMDMCWlbVIgzqAhhjiBzSbAIwBtZQAXAe1ADcbE0TwrRIb2qALSrvgCdQAd2wBPHIJpRoAOgBQIUAGU6AYx7ZIBZARyh8RbFXCD+8A-CjqqiGpGpb24DKmp1zbM48bNQALQAfKDmpoKOsgjI7phKYNigJNjqANZigtBY6jQAtgAO2IhUZCSUmtoEsIoKOZAY7OYYNGT0sAAqNABqTGgAvDgY4pDqoAAUAJQAXKAACtK5mLAAPH7QIX0hAN4AvgDctQ4NobDNrR00AKow0Sj9g8PqkzPzeUvLUUh3G9sKoP8ncKRG5fVAHfYKQ71RqwIHtcT5WAAQTwsEEdiOoAGuEe4wmWN+ANAFHY2BmawORMS2LExROZzanR6zEmlIB1JwtJhDIu1zgoOgrMhRJJSRmnxi0DZ-xIWM5RW5LUZ3V6QqJsppCvpSt5IMlrNACh2kNF2EEBHazgAogAPRG2VDM6DwxEzSYEuYLd5rIIHM0W6122AO6BOl2UAZNHVM3p+82Wky2+1UR29cNyqPnTp825gk2wqQBxNBkM5gXht34zaet5uD5675xotUJPBlPQMuS9OR07RlXMJsJlsl9udu7d7VZq4NvNAA

File: packages/calling-stateful-client/review/calling-stateful-client.api.md [hunk 2]
```diff
@@ -201,6 +203,12 @@ export interface TranscriptionCallFeature {
     isTranscriptionActive: boolean;
 }
 
+// @public (undocumented)
```
Comment (prprabhu-ms) [inline packages/calling-stateful-client/review/calling-stateful-client.api.md:206]: Please document new public type.

File: packages/calling-stateful-client/review/calling-stateful-client.api.md [hunk 3]
```diff
@@ -201,6 +203,12 @@ export interface TranscriptionCallFeature {
...
+export type VideoStreamRendererAndView = {
```
Comment (prprabhu-ms) [inline packages/calling-stateful-client/review/calling-stateful-client.api.md:207]: I'm uncomfortable naming a type for the fields it contains (can never add a field). `CreateViewResult` would be a better name.

File: packages/storybook/stories/Stateful Client/OverviewStateful.stories.mdx [hunk 4]
```diff
@@ -81,7 +81,7 @@ The device manager allows developers to access the local video stream to render
 const cameras = await deviceManager.getCameras();
 const camera = cameras[0];
 const localCameraStream = { source: camera, mediaStreamType: 'Video' } as LocalVideoStreamState;
-const view = statefulClient.createView(localCameraStream);
```
Comment (prprabhu-ms) [inline packages/storybook/stories/Stateful Client/OverviewStateful.stories.mdx:84]: So this story was... simply wrong before this PR.

Comment (JamesBurnside) [reply packages/storybook/stories/Stateful Client/OverviewStateful.stories.mdx:84]: Yep lol

Comment (JamesBurnside) [reply packages/storybook/stories/Stateful Client/OverviewStateful.stories.mdx:84]: Really we need to be able to include sub snippets of snippets so the few lines we want to show have been compile-checked

Comment (JamesBurnside) [reply packages/storybook/stories/Stateful Client/OverviewStateful.stories.mdx:84]: Or perhaps a snippet view that can only show a few lines and then has an "expand" button that shows the whole snippet, imports etc. included

File: packages/calling-stateful-client/review/calling-stateful-client.api.md (hunk 3 above)
Comment (JamesBurnside) [reply packages/calling-stateful-client/review/calling-stateful-client.api.md]: Good point, yeah I agree, updated

**Review by prprabhu-ms — APPROVED** [review]

**Review by PorterNan — APPROVED** [review]

**Pull request merged**